src/scene/vcModel.cpp

```diff
@@ -758,10 +689,33 @@ udDouble3 vcModel::GetLocalSpacePivot()
 
 udDouble4x4 vcModel::GetWorldSpaceMatrix()
 {
-  return m_sceneMatrix;
+  return udDouble4x4::translation(m_pivot) * udDouble4x4::rotationQuat(m_orientationCurrent, m_positionCurrent) * udDouble4x4::scaleUniform(m_scaleCurrent.x) * udDouble4x4::translation(-m_pivot);
 }
 
 vcGizmoAllowedControls vcModel::GetAllowedControls()
 {
   return vcGAC_AllUniform;
 }
+
+void vcModel::SetTransforms(const udDouble4x4 &matrix)
+{
+  (udDouble4x4::translation(-m_pivot) * matrix * udDouble4x4::translation(m_pivot)).extractTransforms(m_positionCurrent, m_scaleCurrent, m_orientationCurrent);
+}
+
+void vcModel::SaveTransformsToNode(vcProject * pProject, const udGeoZone &defaultZone)
```

Comment:
I think you have your visual studio settings wrong, your `*` always ends up in the middle instead of on the right

Reply:
Gah, old habits die hard...